resources/qml/PrintMonitor.qml

```diff
@@ -20,49 +20,462 @@ Column
         simpleNames: true
     }
 
-    Item
+    Rectangle
     {
-        width: base.width - 2 * UM.Theme.getSize("default_margin").width
-        height: childrenRect.height + UM.Theme.getSize("default_margin").height
-        anchors.left: parent.left
-        anchors.leftMargin: UM.Theme.getSize("default_margin").width
+        id: connectedPrinterHeader
+        width: parent.width
+        height: childrenRect.height + UM.Theme.getSize("default_margin").height * 2
+        color: UM.Theme.getColor("setting_category")
 
         Label
         {
+            id: connectedPrinterNameLabel
+            text: printerConnected ? connectedPrinter.name : catalog.i18nc("@info:status", "No printer connected")
```

Comment:
It seems to be defined in several places:
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/resources/qml/Sidebar.qml#L32
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/plugins/UM3NetworkPrinting/UM3InfoComponents.qml#L14
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/resources/qml/Preferences/MachinesPage.qml#L140
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/resources/qml/Menus/NozzleMenu.qml#L16
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/resources/qml/MonitorButton.qml#L18
* https://github.com/Ultimaker/Cura/blob/feature_preheat_bed/resources/qml/Menus/MaterialMenu.qml#L16

The one in Sidebar seems to be most closely related. I think it's that one, but I'm not sure how that property ends up here then. I'll refactor this to make it not use this property any more.